packages/phoenix-event-display/src/managers/three-manager/index.ts

```diff
@@ -624,7 +639,7 @@ export class ThreeManager {
           case 'KeyC': // shift + "c"
             this.setClipping(!this.rendererManager.getLocalClipping());
             if (this.rendererManager.getLocalClipping()) {
-              this.rotateClipping(180);
+              this.setClippingAngle(180, 0);
```

Comment:
I think this will now become `this.setClipping(0, 180)`.

Reply:
Yes, thanks for pointing it out.